package.json

```diff
@@ -7,6 +7,7 @@
   "license": "GPL-2.0-only",
   "dependencies": {
     "@discourse/itsatrap": "^2.0.10",
+    "@discourse/moment-timezone-names-translations": "^1.0.0",
```

Comment:
You'll need to add the same to `app/assets/javascripts/select-kit/package.json`

Reply:
Are you sure this is needed? We only serve these via Rails afaict.

Reply:
Oh, ok. So we have some rails-only js dependencies? 👀